### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-shubham-morankar**
PMDB Link: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=23516

Mantis Link: https://mantis.fortinet.com/bug_view_page.php?bug_id=0866401

Mantis Summary: Missing Endpoints for Application list and info

#### Enhancements:

- New actions are added:
    - Get Application List
    - Get Application Defects List
    
#### Fix:

- Updated "Application Name" parameter with "Analysis Code" for "Get Application Analysis" operation.

#### UTCs:

- [x] Connector installation verified.
- [x] Connector image verified.
- [x] Actions and Playbooks list verified.
- [x] Get Application List action verified (Sanity Test).
- [x] Get Application Defects List action verified (Sanity Test).
- [x] Get Application Analysis action verified (Sanity Test).

### CODE REVIEW

File: kiuwan/release_notes.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #### What's Added
 
-- Added 'Corporate Domain ID' as a configuration parameter.
+- Added the following operations and playbooks
```
Comment (cs-suyog-jinturkar) [inline kiuwan/release_notes.md]: Suggestion: Added the following operations and respective sample playbooks

File: kiuwan/release_notes.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+    - Get Application List
+    - Get Application Defects List
 
 #### What's Fixed
 
-- Fixed a bug where check health was failing with correct configuration.
\ No newline at end of file
+- Updated "Application Name" parameter with "Analysis Code" for "Get Application Analysis" operation.
```
Comment (cs-suyog-jinturkar) [inline kiuwan/release_notes.md]: Suggestion: Updated "Get Application Analysis" action with "Application Name" parameter replaced with "Analysis Code".

File: kiuwan/operations.py [hunk 3]
```diff
@@ -89,6 +89,65 @@ def check_payload(payload):
...
+    endDateAnalysis = params.get('endDateAnalysis')
+    if endDateAnalysis:
+        endDateAnalysis = convert_datetime_in_format(endDateAnalysis)
+    orderBy = params.get('orderBy')
+    if orderBy:
+        orderBy = sort_dict.get(orderBy)
+    payload = {
+        'applicationName': params.get('app_name'),
+        'activityInfo': params.get('activityInfo'),
+        'initDateAnalysis': initDateAnalysis,
```
Comment (cs-suyog-jinturkar) [inline kiuwan/operations.py:116]: You can create a common function to build required params from available params and use same in diff functions/actions.

Comment (cs-shubham-morankar) [reply kiuwan/operations.py:116]: Many of the keys name are different for all operations, that's the reason we cannot use same function for payload.

**Pull request merged**